apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -199,30 +246,42 @@ export default class AnimationPickerBody extends React.Component {
       onAnimationSelectionComplete
     } = this.props;
 
+    let paddingOffset = this.state.headerOffset;
+    if (searchQuery !== '' || categoryQuery !== '') {
+      paddingOffset += this.state.categoryHeight + this.state.footerHeight;
+    }
+
     // Display second "Done" button. Useful for mobile, where the original "done" button might not be on screen when
     // animation picker is loaded. 600 pixels is minimum height of the animation picker.
     const shouldDisplaySecondDoneButton =
       this.multiSelectEnabled_ && isMobileDevice();
 
+    if (shouldDisplaySecondDoneButton) {
+      paddingOffset += this.state.footerHeight;
+    }
     return (
-      <div style={{marginBottom: 10}}>
+      <div style={{height: '100%'}}>
         {shouldDisplaySecondDoneButton && (
           <Button
             text={msg.done()}
             onClick={onAnimationSelectionComplete}
             color={Button.ButtonColor.orange}
           />
         )}
-        <h1 style={dialogStyles.title}>{msg.animationPicker_title()}</h1>
+        <h1 style={dialogStyles.title} ref="animationPicker_title">
```

Comment:
this way of creating/managing refs is deprecated. the first example in the [react docs](https://reactjs.org/docs/refs-and-the-dom.html) shows the preferred method in class components